widget/playground/src/components/ColorItemPicker/ColorItemPicker.tsx

```diff
@@ -0,0 +1,37 @@
+import type { PropTypes } from './ColorItemPicker.types';
+
+import { Button, Typography } from '@rango-dev/ui';
+import React from 'react';
+
+import { Container } from './ColorItemPickerstyles';
+import { ColorPicker } from '../ColorPicker';
+
+function ColorItemPicker(props: PropTypes) {
+  const { label, placeholder, color, onChangeColor, onReset, resetDisable } =
+    props;
+
+  return (
+    <Container>
+      <Typography variant="body" size="small" color="neutral900">
+        {label}
+      </Typography>
+      <ColorPicker
+        place="top"
+        placeholder={placeholder}
+        color={color}
+        onChangeColor={onChangeColor}
+      />
```

Comment:
Could we disable color picker when this item is disabled? :thinking: 

![Recording 2023-10-30 at 16 11 30](https://github.com/rango-exchange/rango-client/assets/99180934/3dc2a680-36aa-4e38-965e-37e08d5aa522)